GATK/4.2.0.0/SelectVariants.nf

```diff
@@ -0,0 +1,55 @@
+process SelectVariantsSample {
+    tag {"GATK SelectVariantsSample ${analysis_id} - ${sample_id}"}
+    label 'GATK_4_2_0_0'
+    label 'GATK_4_2_0_0_SelectVariantsSample'
+    container = 'broadinstitute/gatk:4.2.0.0'
+    shell = ['/bin/bash', '-euo', 'pipefail']
+
+    input:
+        tuple(val(analysis_id), path(vcf_file), path(vcf_idx_file), sample_id)
+
+    output:
+        // simpleName instead of baseName, in case of .vcf.gz
+        tuple(
+            val(sample_id), 
+            path("${sample_id}_${vcf_file.simpleName}.vcf.gz"), 
+            path("${sample_id}_${vcf_file.simpleName}.vcf.gz.tbi"), 
+            emit: vcf_file
+        )
+
+    script:
+        """
+        gatk --java-options "-Xmx${task.memory.toGiga()-4}G" SelectVariants \
+        --reference ${params.genome} \
+        -variant ${vcf_file} \
```

Comment:
According to docs it is `-V` or `--variant`. 
https://gatk.broadinstitute.org/hc/en-us/articles/4409897176987-SelectVariants#--variant